File: library/src/main/java/nz/co/cic/mdns/MDNSScanResultListener.java
package nz.co.cic.mdns;

/**
 * Created by dipshit on 3/03/17.
 */

public interface MDNSScanResultListener {

    void onResult(MDNSAdvertisement mdnsAdvertisement);
    void onLost(MDNSAdvertisement mdnsAdvertisement);
}

File: library/src/main/java/nz/co/cic/mdns/MDNSScanResultListener.kt
package nz.co.cic.mdns

/**
 * Created by dipshit on 3/03/17.
 */

interface MDNSScanResultListener {

    fun onResult(mdnsAdvertisement: MDNSAdvertisement)
    fun onLost(mdnsAdvertisement: MDNSAdvertisement)
}
